pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/QueryOp.java

```diff
@@ -55,7 +72,97 @@ public void setExpectedResultsFileName(String expectedResultsFileName) {
 
   @Override
   boolean runOp() {
-    System.out.println("Verifying queries in " + _queryFileName + " against results in " + _expectedResultsFileName);
-    return true;
+    System.out.printf("Verifying queries in %s against results in %s\n", _queryFileName, _expectedResultsFileName);
```

Comment:
Why not use the `LOGGER` you specified in Line 41?

Reply:
Just  wanner make it consistence with StreamOp and TableOp, basically every Op will print out such a info message in console.

Reply:
For now, it is ok.  For some reason , logs don't show up when we run the ops via command-line. Eventually we will remove the sout or keep a small number of them if needed.

Reply:
Then why use `printf` instead of `println` here?

Reply:
As suggested, please change this to println instead of printf. thanks